homeassistant/components/trace/__init__.py

```diff
@@ -216,15 +218,15 @@ class ActionTrace(BaseTrace):
 
     def __init__(
         self,
-        item_id: str,
-        config: dict[str, Any],
-        blueprint_inputs: dict[str, Any],
+        item_id: str | None,
+        config: dict[str, Any] | None,
+        blueprint_inputs: dict[str, Any] | None,
```

Comment:
I've enabled `None` here for `config` and `blueprint_inputs`
I couldn't find a reason for them to be compulsory, and it helps with #78368